dbms/src/Dictionaries/IDictionary.h

```diff
@@ -47,7 +45,7 @@ struct IDictionaryBase : public IExternalLoadable
 
     virtual BlockInputStreamPtr getBlockInputStream(const Names & column_names, size_t max_block_size) const = 0;
 
-    bool supportUpdates() const override { return !isCached(); }
+    bool supportUpdates() const override { return true; }
```

Comment:
Ok but the name is weird (because all dictionaries are updated anyway).
If I remember correctly, this method is about support for selective updates.